www/src/css/headline.css

```diff
@@ -8,7 +8,7 @@
   margin:0 0 0 10px
 }
 
-.b-headline .b-headline__store-link {
+.region-installed .b-headline__store-link {
```

Comment:
This is the issue mentioned in https://github.com/snapcore/snapweb/pull/22#discussion_r70395571